pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/PinotHelixResourceManager.java

```diff
@@ -3923,18 +3935,21 @@ public Map<String, List<InstanceInfo>> getTableToLiveBrokersMapping() {
         .collect(Collectors.toMap(InstanceConfig::getInstanceName, Function.identity()));
 
     Map<String, List<InstanceInfo>> result = new HashMap<>();
+    if (optionalTableName.isPresent()) {
+      String tableName = optionalTableName.get();
```

Comment:
This `tableName` can be raw (without type suffix). The returned map should contain the actual table name with type suffix